CoroutineTemplate/build.gradle

```diff
@@ -18,14 +18,19 @@ buildscript {
 
         constraint_layout_version = '2.0.0-alpha3'
 
-        hilt_version = '2.33-beta'
+        hilt_version = '2.35'
```

Comment:
I think we can upgrade this to `2.37` 🙏🏼

Reply:
@chornerman So fast Let me check that🙏